tests/release/pipelines/fbc_release.go

```diff
@@ -267,6 +326,8 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 			"buildTimeoutSeconds":             "480",
 			"hotfix":                          hotfix,
 			"issueId":                         issueId,
+			"productName":                     productName,
```

Comment:
@theflockers having these keys in the data field with empty string value is okay for the tasks? Just wanted to check

Reply:
If `preGA` is set and `productName` or `productVersion` has the length of zero, the pipelinerun will fail saying that these values are required. But only if `preGA` is set. If it is not set, the value is irrelevant.

Reply:
Shouldn't this RPA have `preGA` set then for the `preGA` usecase? Or maybe I am misunderstanding where it is set

Reply:
@theflockers ,  is the `preGA` parameter need to be added and tested?  I didn't notice `preGA` can be set in RPA before.

Reply:
Indeed. This passed unnoticed by me. Pre-GA now requires the `preGA` flag set to true so I can read the `productName` and `productVersion`. Otherwise these values will be ignored.

Reply:
Thanks.  Added `preGA` parameter in RPA. Can you please review it again?